.github/workflows/test-pg_search.yml

```diff
@@ -225,3 +225,8 @@ jobs:
         if: steps.check_skip.outputs.skip_remaining_steps != 'true' && github.base_ref == 'main'
         working-directory: pg_search/
         run: cargo clean && cargo pgrx package --features icu
+
+      # Print postgres logs
+      - name: Postgres logs
```

Comment:
If we're putting this here, can we also put in the `test-pg_lakehouse.yml`?

maybe just do `name: Print Postgres Logs` instead of the extra comment? And can remove the `#run even on failure`

Reply:
I'm just debugging now, let me know if you do want to keep it.

Reply:
Aaah, sorry. If you don't think we need to keep it you can remove it. I thought you wanted it there all the time